bson/src/main/org/bson/codecs/pojo/PojoBuilderHelper.java

```diff
@@ -53,88 +46,104 @@
         }
```

Comment:
I'll double check, previously classmate would NPE reflecting on an Object.class but I removed the ObjectCodec and must have removed the roundtrip test case. 

I'll add a test case for it